render.js

```diff
@@ -0,0 +1,64 @@
+
+function render(dataOrString, renderer, configuration) {
+
+    /* Convert a string to the BCD data */
+    let data = undefined;
+    if (typeof dataOrString === 'string' || dataOrString instanceof String) {
+        const dataParts = dataOrString.split('.');
+        data = require('.');
+        dataParts.forEach((elem) => {
+            data = data[elem] || {};
+        });
+    } else {
+        data = dataOrString;
+    }
+    return renderer(data, configuration);
+}
+
+function usage() {
+    const usageString = `Render a feature as an HTML table.
+
+Usage:
+ npm run render <featurePath> [depth] [aggregateMode]
+
+Options:
+
+ featurePath: Dotted path to feature
+ depth: Traversal depth
+ aggregateMode: skip "Basic Support" row
```

Comment:
Aggregate mode could be a different renderer instead of a general `render` option.

Reply:
yes, aggregate mode is a feature of the old compat table renderer, which I plan to replace in the next PR. It can go now.